### PR TITLE
Fix Thanos Store startup

### PR DESCRIPTION
**Pull request by povilasv**

### CODE REVIEW

File: base/local/thanos-store.yaml [hunk 1]
```diff
@@ -54,19 +55,19 @@ spec:
             path: /metrics
             port: http
             scheme: HTTP
-          periodSeconds: 30
+          periodSeconds: 20
           successThreshold: 1
-          failureThreshold: 5
+          failureThreshold: 100
```
Comment (ffilippopoulos) [inline base/local/thanos-store.yaml]: :astonished:

Comment (povilasv) [reply base/local/thanos-store.yaml]: why not?

Comment (ffilippopoulos) [reply base/local/thanos-store.yaml]: just amazed by how much you had to loose this in order to let start

Comment (povilasv) [reply base/local/thanos-store.yaml]: I just didnt want to play around with the value, so  figured why not make it big

Comment (ffilippopoulos) [reply base/local/thanos-store.yaml]: wouldn't that give a period of 2000 seconds where liveness probe will fail but kube won't care about it? so a good half hour.

**Comment by povilasv** [comment]
@ffilippopoulos changed 100 -> 25 tries. 25 * 20 s, ~8 mins which is fine for fat thanos-store gw

**Pull request merged**